### PR TITLE
add Quarkus pet store sample.

### PR DESCRIPTION
**Pull request by bnusunny**
Add sample for quarkus pet store. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: samples/quarkus/pet-store/pom.xml [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.amazon.quarkus.demo</groupId>
```
Comment (sapessi) [inline samples/quarkus/pet-store/pom.xml]: Should be com.amazonaws.serverless.sample and the artifact serverless-quarkus-example to follow the same pattern we use in the [regular samples](https://github.com/awslabs/aws-serverless-java-container/blob/master/samples/jersey/pet-store/pom.xml).

File: samples/quarkus/pet-store/sam.native.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    Api:
+      EndpointConfiguration: REGIONAL
+      BinaryMediaTypes:
+        - "*/*"
+
+  Resources:
+    PetStoreNativeFunction:
+      Type: AWS::Serverless::Function
+      Properties:
+        Handler: not.used.in.provided.runtimei
```
Comment (sapessi) [inline samples/quarkus/pet-store/sam.native.yaml]: one "i" too many at the end

File: samples/quarkus/pet-store/sam.native.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        Runtime: provided
+        CodeUri: target/function.zip
+        MemorySize: 128
+        Policies: AWSLambdaBasicExecutionRole
+        Tracing: Active
+        Timeout: 15
+        Environment:
+          Variables:
+            DISABLE_SIGNAL_HANDLERS: true
```
Comment (sapessi) [inline samples/quarkus/pet-store/sam.native.yaml]: What does this do? Is it worth mentioning it in the readme?

File: samples/quarkus/pet-store/src/main/resources/application.properties [hunk 4]
```diff
@@ -0,0 +1,4 @@
+quarkus.native.container-build=true
+quarkus.native.container-runtime=docker
+quarkus.native.builder-image=quay.azk8s.cn/quarkus/ubi-quarkus-native-image:19.2.1
```
Comment (sapessi) [inline samples/quarkus/pet-store/src/main/resources/application.properties]: The Quarkus repo says we should be [pulling from quay.io](https://github.com/quarkusio/quarkus-images) - what is the azk8s.cn? Also, are we sure that this image will always generate executables for the correct target platform/libc? In the micronaut sample we fix on the [Docker image for Lambda's AMI](https://github.com/awslabs/aws-serverless-java-container/blob/master/samples/micronaut/pet-store/Dockerfile#L1)

File: samples/quarkus/pet-store/pom.xml (hunk 1 above)
Comment (bnusunny) [reply samples/quarkus/pet-store/pom.xml]: updated.

File: samples/quarkus/pet-store/sam.native.yaml (hunk 2 above)
Comment (bnusunny) [reply samples/quarkus/pet-store/sam.native.yaml]: corrected.

File: samples/quarkus/pet-store/sam.native.yaml (hunk 3 above)
Comment (bnusunny) [reply samples/quarkus/pet-store/sam.native.yaml]: This is mentioned in Quarkus doc and a github issue. 
https://github.com/quarkusio/quarkus/issues/4262#issuecomment-537599842

It seems custom runtime does not allow the syscalls required by the GraalVM signals implementation. 

I added this in the readme.

File: samples/quarkus/pet-store/src/main/resources/application.properties (hunk 4 above)
Comment (bnusunny) [reply samples/quarkus/pet-store/src/main/resources/application.properties]: Quay.io is not accessible from mainland China. azk8s.cn is a local mirror for quay.io. I changed it to quay.io. 

This image works great to build native exectuabels for lambda custom time.

**Pull request merged**